source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1096,9 +1096,12 @@ void Filter::sendImmediateResponse(const ImmediateResponse& response) {
 
   sent_immediate_response_ = true;
   ENVOY_LOG(debug, "Sending local reply with status code {}", status_code);
-  const auto details = StringUtil::replaceAllEmptySpace(response.details());
+  const std::string prefix =
+      fmt::format("immediate_response_from_ext_proc[{}]", encoder_callbacks_->filterConfigName());
```

Comment:
I suggest direct_response_from_ext_proc.. I know immediate response is in the proto but the router uses direct_response in its API and we also want to use the same model in ext_authz for its direct response.

Reply:
I am not sure if we should exactly follow the name from router API. Here is `direct/immediate_reponse_from_ext_proc` that is different from router's one anyway.

I don't have a strong opinion on this though.